### DISCUSSION TITLE
June 24

### DISCUSSION THREAD

**Discussion by TimZiemer** [ifsm/pymus#23] [Announcements]
As  I will be on a conference next week, we will have a challenge rather than a seminar: In the codebase of the main branch, you will find the "Chellenge" folder including a large csv file with random numbers (numbers.csv). Your challenge will be to
1.) import the data from the file
2.) implement your own sorting algorithm to sort them (please do not use a pre-defined sorting algorithm)
3.) measure and report the computation time.

You can use the github discussions function to post relevant literature, ask for help or provide help to others.

**Comment by barisusseli** [comment]
Should we upload our algorithms here in the discussion or push them to a certain branch?

**Reply by TimZiemer** [reply]
Use your name as a filename and merge it into the main branch under 

https://github.com/ifsm/pymus/tree/main/Challenge
